deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/KerasSequentialModel.java

```diff
@@ -0,0 +1,172 @@
+        importTrainingConfiguration(trainingConfigJson);
+    }
+
+    /**
+     * Constructor for Sequential model and training configuration JSON strings and map containing weights.
+     *
+     * @param modelConfigJson       model configuration JSON string
+     * @param trainingConfigJson    training configuration JSON string
+     * @param weights               map from layer to parameter to weights
+     * @throws IOException
+     */
+    public KerasSequentialModel(String modelConfigJson, String trainingConfigJson, Map<String, Map<String,INDArray>> weights)
+            throws IOException, InvalidKerasConfigurationException {
+        this(modelConfigJson);
+        importTrainingConfiguration(trainingConfigJson);
+        copyWeightsToModel(weights);
+    }
+
+    /**
+     * Configure a MultiLayerConfiguration from this Keras Sequential model configuration.
+     *
+     * @return          MultiLayerConfiguration
+     */
+    public MultiLayerConfiguration getMultiLayerConfiguration()
+            throws InvalidKerasConfigurationException, UnsupportedKerasConfigurationException {
+        if (!this.className.equals(MODEL_CLASS_NAME_SEQUENTIAL))
+            throw new InvalidKerasConfigurationException("Keras model class name " + this.className + " incompatible with MultiLayerNetwork");
+        if (this.inputLayerNames.size() != 1)
+            throw new InvalidKerasConfigurationException("MultiLayeNetwork expects only 1 input (found " + this.inputLayerNames.size() + ")");
+        if (this.outputLayerNames.size() != 1)
+            throw new InvalidKerasConfigurationException("MultiLayeNetwork expects only 1 output (found " + this.outputLayerNames.size() + ")");
+
+        NeuralNetConfiguration.Builder modelBuilder = new NeuralNetConfiguration.Builder();
+        NeuralNetConfiguration.ListBuilder listBuilder = modelBuilder.list();
+
+        /* Add layers one at a time. */
+        int layerIndex = 0;
+        for (String layerName : this.layerNamesOrdered) {
+            KerasLayer layer = this.layers.get(layerName);
+            if (layer.isDl4jLayer()) { // Ignore "preprocessor" layers for now
+                listBuilder.layer(layerIndex++, layer.getDl4jLayer());
+            }
+        }
+
+        InputType inputType = inferInputType(this.inputLayerNames.get(0));
+        if (inputType != null)
+            listBuilder.setInputType(inputType);
+
+        /* Handle truncated BPTT:
+         * - less than zero if no recurrent layerNamesOrdered found
+         * - greater than zero if found recurrent layer and truncation length was set
+         * - equal to zero if found recurrent layer but no truncation length set (e.g., the
+         *   model was built with Theano backend and used scan symbolic loop instead of
+         *   unrolling the RNN for a fixed number of steps.
+         *
+         * TODO: should we not throw an error for truncatedBPTT==0?
+         */
+        if (this.truncatedBPTT == 0)
```

Comment:
Yeah basically this raises the question of what the proper behavior should be when trying to import a Keras-defined RNN that does not use truncated backprop (not unusual with Theano backend). Should we
  - throw an error (which makes problem obvious to user, who can then go back and change their Keras configuration)?
  - choose some reasonable default BPTT value ourselves and maybe mention this in a warning?
  - NOT set truncated BPTT in the DL4J model configuration? What exactly would the behavior be for this (I haven't tried it in DL4J yet)?